Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM nixos/nix
+ADD . .
+RUN nix-channel --update
+RUN nix-shell --command ./scripts/integrationtest.sh
```

Comment:
This dockerfile doesn't have a `CMD` or `ENTRYPOINT` so I don't think you can really run this multiple times. When you build the image with `docker build` it will run the integration tests *at build time*. When you run the container with `docker run` then nothing happens since there's no `CMD` or `ENTRYPOINT` ...